docs/user-guide/commands/display.md

```
@@ -0,0 +1,69 @@
+
+**When to use it:**
+
+- Quickly reviewing a config backup without opening a separate viewer
+- Spot-checking a specific section (e.g., firewall rules or interfaces) during troubleshooting
+- Verifying what a config contains before running a full conversion or audit
+- Reviewing configs over SSH where you cannot easily open generated files
+
+## Usage
+
+```
+opndossier display [flags] <config.xml>
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)